### PR TITLE
pdnsd: support pdnsd

### PR DESCRIPTION
**Pull request by peterwillcn**
Signed-off-by: Xiaobo Tian <peterwillcn@gmail.com>

Compile tested: OpenWrt master  x86_64 ramips tests done
Run tested:  OpenWrt master  x86_64 ramips tests done

Description:
  Add support pdnsd

### CODE REVIEW

**Comment by diizzyy** [comment]
Please elaborate why non of the packaged counterparts doesn't fit your need

File: net/pdnsd/Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+PKG_NAME:=pdnsd
+PKG_VERSION:=1.2.9a
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/peterwillcn/pdnsd.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=a8e46ccba7b0fa2230d6c42ab6dcd92926f6c21d
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```
Comment (neheb) [inline net/pdnsd/Makefile]: Remove. PKG_SOURCE_PROTO already does this.

File: net/pdnsd/Makefile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+PKG_MIRROR_HASH:=ca16c16d3d94477414cb412a260ff648937f2d6bcc2c1b7cd75ff8a41afbe9db
+# CMAKE_INSTALL:=1
```
Comment (neheb) [inline net/pdnsd/Makefile]: Remove

File: net/pdnsd/Makefile [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  contents are written to hard disk on exit) that is designed to cope with
+  unreachable or down DNS servers (for example in dial-in networking).
+
+  pdnsd can be used with applications that do dns lookups, eg on startup, and
+  can't be configured to change that behaviour, to prevent the often
+  minute-long hangs (or even crashes) that result from stalled dns queries.
+endef
+
+TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
+#TARGET_CFLAGS += -ggdb3
```
Comment (neheb) [inline net/pdnsd/Makefile]: Remove

File: net/pdnsd/Makefile [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+#CMAKE_OPTIONS += -DDEBUG=1
```
Comment (neheb) [inline net/pdnsd/Makefile]: Remove

File: net/pdnsd/files/pdnsd.init [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+       gen_cache
+
+       $DAEMON --daemon -p $PID_FILE
+       echo " ."
+}
+
+stop() {
+       echo -n "Stopping $DESC: $NAME"
+       kill `cat $PID_FILE` > /dev/null 2>&1
```
Comment (neheb) [inline net/pdnsd/files/pdnsd.init]: run through shellcheck. This will definitely trigger a warning.

**Comment by diizzyy** [comment]
@peterwillcn 
Ping

**Comment by peterwillcn** [comment]
close this, open new pr https://github.com/openwrt/packages/pull/9481

**Comment by diizzyy** [comment]
Please don't do that next time,

**Pull request closed without merging**